src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/SmokeTestSuite.java

```diff
@@ -21,31 +19,29 @@
 import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
-@RunWith(SpringRunner.class)
-@SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
 @TestPropertySource("classpath:application.properties")
-public abstract class SmokeTestSuite {
+class SmokeTestSuite {
 
     @Value("${test-url}")
-    protected String testUrl;
+    String testUrl;
```

Comment:
Is this used by some other class in the package or is this is missed ?

Reply:
Yes, used in another class. would've failed the `./gradlew check` step otherwise